src/storage/storage.cc

```diff
@@ -98,6 +98,7 @@ void Storage::CloseDB() {
   db_->SyncWAL();
   rocksdb::CancelAllBackgroundWork(db_.get(), true);
   for (auto handle : cf_handles_) db_->DestroyColumnFamilyHandle(handle);
+  shared_block_cache_->DisownData();
```

Comment:
Should we add another flag for this? Since I don't know CloseDb in testing or other case would checking leak

Reply:
ok then.

`fast_shutdown`?

Reply:
this is ok to me